CONTRIBUTING.md

```diff
@@ -10,6 +10,7 @@
 * Code the algorithm/DS following the styleguide defined below.
 * Send a PR. 
 * Be sure to not include any compiled binaries in the patch.
+* While sending a PR make sure you follow one PR per issue rule.
```

Comment:
@aviaryan I had started #107 with the name one issue per PR but was changed later by @singhpratyush sir.
So what should it be? your views.